### PR TITLE
Adds sign up and sign in

### PR DESCRIPTION
**Pull request by DCox2016**
Home page with sign in and sign up
Screen Shot 2019-04-24 at 2 10 30 PM
User signed in page
Screen Shot 2019-04-24 at 2 10 41 PM
User sign up page
Screen Shot 2019-04-24 at 2 10 51 PM

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-const routes: Routes = [];
+const routes: Routes = [
+  {path: 'signup', component: SignupComponent },
+  {path: 'signin', component: HomeComponent}
+];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
+  imports: [ RouterModule.forRoot(routes)],
```
Comment (cpyle0819) [inline src/app/app-routing.module.ts]:
```suggestion
  imports: [ RouterModule.forRoot(routes) ],
```

File: src/app/dashboard/dashboard.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  let fixture: ComponentFixture<DashboardComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ DashboardComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
```
Comment (cpyle0819) [inline src/app/dashboard/dashboard.component.spec.ts]: I wouldn't have two beforeEach functions. These should be consolidated.

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div id="isSignedIn">
+  <a routerLink="/signOut">Sign out</a>&nbsp;
```
Comment (cpyle0819) [inline src/app/home/home.component.html]:
```suggestion
  <a routerLink="/signOut">Sign out</a>
```

File: src/app/login/login.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<div id="signUpsignin">
+<form align="right">
+  Username:
```
Comment (cpyle0819) [inline src/app/login/login.component.html]: Labels should be used here like you're using for the signup component

File: src/app/signup/signup.component.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    <input type="text" placeholder="Enter Email" name="email" required><br>
+
+    <label for="psw"><b>Password</b></label>
+    <input type="password" placeholder="Enter Password" name="psw" required><br>
+
+    <label for="psw-repeat"><b>Repeat Password</b></label>
+    <input type="password" placeholder="Repeat Password" name="psw-repeat" required><br>
+
+    <label>
+      <input type="checkbox" checked="checked" name="remember" style="margin-bottom:15px"> Remember me
```
Comment (cpyle0819) [inline src/app/signup/signup.component.html]: Don't use inline styles.

File: src/app/signup/signup.component.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    </label>
+
+    <p>By creating an account you agree to our <a href="#" style="color:dodgerblue">Terms & Privacy</a>.</p>
```
Comment (cpyle0819) [inline src/app/signup/signup.component.html]: Don't use inline styles.

File: src/app/dashboard/dashboard.component.spec.ts (hunk 2 above)
Comment (DCox2016) [reply src/app/dashboard/dashboard.component.spec.ts]: removed generated by default when using ng g component

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (DCox2016) [reply src/app/app-routing.module.ts]: added space

File: src/app/home/home.component.html (hunk 3 above)
Comment (DCox2016) [reply src/app/home/home.component.html]: removed non blank space

File: src/app/login/login.component.html (hunk 4 above)
Comment (DCox2016) [reply src/app/login/login.component.html]: added labels

File: src/app/signup/signup.component.html (hunk 5 above)
Comment (DCox2016) [reply src/app/signup/signup.component.html]: removed inline styles

File: src/app/signup/signup.component.html (hunk 6 above)
Comment (DCox2016) [reply src/app/signup/signup.component.html]: removed inline styles

**Comment by DCox2016** [comment]
Screen Shot 2019-04-28 at 4 08 53 PM

File: package.json [hunk 7]
```diff
@@ -19,30 +19,45 @@
...
     "karma-jasmine": "~1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
+    "lint-staged": "^8.1.5",
+    "prettier": "1.17.0",
     "protractor": "~5.4.0",
     "ts-node": "~7.0.0",
     "tslint": "~5.11.0",
     "typescript": "~3.1.6"
+  },
+  "husky": {
```
Comment (cpyle0819) [inline package.json]: nice

File: src/app/home/home.component.css [hunk 8]
```diff
@@ -0,0 +1,13 @@
+body {
+  background-color: black;
+  border-style: outset;
+  border-color: #bb0a1e;
+}
+body img{
```
Comment (cpyle0819) [inline src/app/home/home.component.css]: you should see if you can configure prettier to do your css files too. I like separating style blocks with new lines, and keeping spaces before opening curly braces.

File: src/app/home/home.component.html [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<div id="isSignedIn">
+  <a routerLink="/signOut">Sign out</a>
+</div>
+<body>
+<img src='../../assets/images/awaywego.png'>
```
Comment (cpyle0819) [inline src/app/home/home.component.html]:
```suggestion
    <img src='../../assets/images/awaywego.png'>
```

**Pull request merged**